src/model/pyproject.toml

```diff
@@ -25,11 +25,10 @@ dependencies = [
     "zstandard>=0.21.0",
     "protobuf==3.20.0",
     "lightning>=2.0.2",
-    "bitsandbytes-cuda110",
     "wandb>=0.15.2",
     "bitsandbytes",
     "deepspeed>=0.9.2",
-    "nlpo3>=1.3.0",
     "peft>=0.3.0",
-    "scipy<2.0.0"
+    "scipy<2.0.0",
+    "tensorboard",
```

Comment:
Version?

Reply:
We can use 
- `^x.x.x` = update to latest version of `x.x.x` except major version change
- `~x.x.x` = update to latest version of `x.x.x` except major and minor version change

Reply:
I think we can use every version.

Reply:
When a major version changes. code usually breaks

Reply:
Done